### ISSUE TITLE
Linting code quoted in `:( )`

### ISSUE THREAD

**Issue by tonyhffong** [tonyhffong/Lint.jl#50]
Apparently there is this very rare edge case

``` julia
ex1 = :( a,b = (1,2) )
ex2 = parse( "a,b = (1,2)" ) # different!
```

**Comment by TotalVerb** [comment]
Perhaps unrelated, but I think we should emit a warning for `parse("blah blah blah")`. This style seems like it's a holdover for people coming from languages where code is represented as a string, and not as a first-class data structure. `:( blah blah blah )` is better because it gives compile-time syntax checking, is more readable, and is probably also faster when run in loops.
